### PR TITLE
#4785 Fix: live test

### LINKED ISSUE #4785: Fix live test

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed live tset

close #4785 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -114,3 +101,11 @@ blocks:
...
           commands:
             - npm run-script test_ci_chrome_consumer_live_gmail
+      epilogue:
+        on_fail:
+          commands:
+            - |
+              if [ -f build/test/test/debugArtifacts/debugHtmlAttachment-0.html ]; then
+                echo "Uploading debug files as job artifacts..."
+                artifact push job build/test/test/debugArtifacts/debugHtmlAttachment-0.html
+              fi
```
Comment (ioanmo226) [inline .semaphore/semaphore.yml:111]: @rrrooommmaaa Any reason why we didn't store `Live test` artifacts before?

Comment (rrrooommmaaa) [reply .semaphore/semaphore.yml:111]: None that I know of.

File: test/source/mock/google/google-data.ts [hunk 2]
```diff
@@ -176,6 +176,18 @@ export class GoogleData {
...
 
+  public static getMockChatPage = (acct: string) => `<!DOCTYPE HTML><html>
+  <body>
+  <div class="gb_Cb">
+    <div class="gb_Ib">${acct}</div>
+  </div>
+  <div class="gb_hb">
+    <div class="gb_lb">Full Name</div>
+  </div>
+  <h1 class="acY">No converation selected</h1>
```
Comment (rrrooommmaaa) [inline test/source/mock/google/google-data.ts:187]: typo: `conversation`

Comment (ioanmo226) [reply test/source/mock/google/google-data.ts:187]: thanks

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (ioanmo226) [reply .semaphore/semaphore.yml:111]: I see

File: test/source/tests/setup.ts [hunk 3]
```diff
@@ -447,6 +447,14 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await gmailPage.notPresent(['@webmail-notification-setup']);
     }));
 
+    ava.default('mail.google.com/chat', testWithBrowser(undefined, async (t, browser) => {
+      const acct = 'ci.tests.gmail@flowcrypt.dev';
+      const settingsPage = await BrowserRecipe.openSettingsLoginButCloseOauthWindowBeforeGrantingPermission(t, browser, acct);
+      await settingsPage.close();
+      const googleChatPage = await openMockGmailPage(t, browser, acct, false, TestUrls.mockGoogleChat());
+      await googleChatPage.notPresent(BrowserRecipe.oldAndNewComposeButtonSelectors); // compose button should not be injected
```
Comment (rrrooommmaaa) [inline test/source/tests/setup.ts:455]: I'm not sure what we're testing with the mock here,
especially `div.z0`. Why would it appear if we don't have such elements in our mock chat page?

Comment (ioanmo226) [reply test/source/tests/setup.ts]: You are right. This. test is indeed not necessary for mock.
Just removed it.

**Review by rrrooommmaaa — APPROVED** [review]

**Pull request merged**